plugins/manipulator.cpp

```diff
@@ -2164,14 +3807,12 @@ void viewscreen_unitlaborsst::render()
     OutputString(10, x, y, Screen::getKeyDisplay(interface_key::CUSTOM_P));
     OutputString(15, x, y, ": Apply Profession ");
     OutputString(10, x, y, Screen::getKeyDisplay(interface_key::CUSTOM_SHIFT_P));
-    OutputString(15, x, y, ": Save Prof. ");
+    OutputString(15, x, y, ": Read Prof. ");
 }
 
 df::unit *viewscreen_unitlaborsst::getSelectedUnit()
 {
-    // This query might be from the rename plugin
```

Comment:
This is relevant - it's saying that the rename plugin could have changed some unit's name, so names should be refreshed.